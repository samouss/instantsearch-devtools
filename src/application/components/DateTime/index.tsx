import React from 'react';
import { format as dateFnsFormat } from 'date-fns';

type Props = {
  value: string | number | Date;
  format: string;
};

const DateTime: React.SFC<Props> = ({ value, format }) => (
  <>{dateFnsFormat(value, format)}</>
);

export default DateTime;
