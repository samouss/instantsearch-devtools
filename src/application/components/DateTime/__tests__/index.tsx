import React from 'react';
import { shallow } from 'enzyme';
import DateTime from '../index';

describe('<DateTime />', () => {
  it('exepct to render', () => {
    const props = {
      value: new Date('01/01/2000'),
      format: 'MM-YYYY',
    };

    const wrapper = shallow(<DateTime {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
