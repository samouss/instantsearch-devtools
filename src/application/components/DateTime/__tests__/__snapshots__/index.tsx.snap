// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DateTime /> exepct to render 1`] = `
<React.Fragment>
  01-2000
</React.Fragment>
`;
